tests/lib/options.js

```diff
@@ -8,20 +8,20 @@
 //------------------------------------------------------------------------------
 
 import assert from "assert";
-import { normalizeOptions } from "../../lib/options.js";
+import { normalizeOptions, getLatestEcmaVersion } from "../../lib/options.js";
 
 
 //------------------------------------------------------------------------------
 // Tests
 //------------------------------------------------------------------------------
 
 describe("normalizeOptions", () => {
-    it("should set ecmaVersion to 5 if it wasn't specified", () => {
+    it("should set ecmaVersion to latestEcmaVersion if it wasn't specified", () => {
         const option = {};
 
         const output = normalizeOptions(option);
 
-        assert.strictEqual(output.ecmaVersion, 5);
+        assert.strictEqual(output.ecmaVersion, getLatestEcmaVersion());
     });
```

Comment:
It might be nice to also add a test like this with `normalizeOptions({ ecmaVersion: "latest" })`, to confirm that "latest" means Espree's latest supported ecmaVersion (i.e., that we're not just passing `"latest"` to Acorn).

Reply:
Good idea. I’ll add that.